.database_generator/mangos_translation/lightshope_merge.py

```
@@ -0,0 +1,201 @@
+
+  cur = target_conn.cursor()
+  cur.execute("SELECT name FROM sqlite_master WHERE type='table' AND name=?", (table,))
+  if not cur.fetchone():
+    print(f"[Lightshope-merge] Missing table {table}, importing full file {sql_path}")
+    _import_sql_file_into_db(target_conn, sql_path)
+    return
+
+  key_col = columns[0]
+  for row in rows:
+    key_val = row[0]
+    cur.execute(f"SELECT 1 FROM {table} WHERE {key_col}=?", (key_val,))
```

Ruff (0.14.6)

Possible SQL injection vector through string-based query construction

(S608)